File: python/nt_toolbox/plot_vf.py
import numpy as np
import matplotlib.pyplot as plt
from numpy import random

def plot_vf(velocities):
    """
        velocities is supposed to be of shape nxnx2
    """
    n = np.shape(velocities)[0]
    u = velocities[:,:,0]
    v = velocities[:,:,1]
    x,y = np.meshgrid(np.arange(n), np.arange(n))
    plt.quiver(x,y,u,v,color="b")
    plt.xlim(0,n)
    plt.ylim(0,n)
    plt.axis("off")
    plt.show()

File: julia/NtToolBox/src/plot_vf.jl
function plot_vf(velocities)
    """
        velocities is supposed to be of shape nxnx2
    """
    n = size(velocities,1)
    u = velocities[:,:,1]
    v = velocities[:,:,2]
    x,y = meshgrid(0:n-1, 0:n-1)
    quiver(x,y,u,v,color="b")
    xlim(0,n)
    ylim(0,n)
    axis("off")
    show()
end
